### PR TITLE
Cc 8

### PR DESCRIPTION
**Pull request by StephanieKirshner**

### CODE REVIEW

File: src/components/SignUpModal/SignInUpModal.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { ModalContainer, Content } from './style';
+import './style.js';
```
Comment (jdickerson-banyan) [inline src/components/SignUpModal/SignInUpModal.jsx:3]: This is not necessary. Remove this line.

File: src/components/SignUpModal/SignInUpModal.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
```
Comment (jdickerson-banyan) [inline src/components/SignUpModal/SignInUpModal.jsx:5]: remove the extra line

File: src/setupTests.js [hunk 3]
```diff
@@ -3,3 +3,4 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
+
```
Comment (jdickerson-banyan) [inline src/setupTests.js:6]: why was this file modified?

**Review by jdickerson-banyan — CHANGES_REQUESTED** [review]
Great job on this so far. Please complete the requested changes and take care of your merge conflicts.

File: README.md [hunk 4]
```diff
@@ -1 +1,2 @@
-Jessica Dickerson - Team Lead
\ No newline at end of file
+Jessica Dickerson - Team Lead
+Stephanie Cook -
```
[the remark this answers is not in the document]
Comment (StephanieKirshner) [reply README.md:2]: fixed

File: src/App.js [hunk 5]
```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
+import ReactDOM from 'react-dom';
```
[the remark this answers is not in the document]
Comment (StephanieKirshner) [reply src/App.js]: done

File: src/App.js [hunk 6]
```diff
@@ -1,23 +1,14 @@
...
+
+import SignUpModal from './components/SignUpModal';
+
```
[the remark this answers is not in the document]
Comment (StephanieKirshner) [reply src/App.js]: done

File: src/App.test.js [hunk 7]
```diff
@@ -1,8 +1,11 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
+
+
```
[the remark this answers is not in the document]
Comment (StephanieKirshner) [reply src/App.test.js:5]: I have no idea. I never touched this page

File: src/components/SignUpModal/index.jsx [hunk 8]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { ModalContainer, Content } from './style';
+import './style.js';
+
```
[the remark this answers is not in the document]
Comment (StephanieKirshner) [reply src/components/SignUpModal/index.jsx]: done

File: src/components/SignUpModal/SignInUpModal.jsx (hunk 1 above)
Comment (StephanieKirshner) [reply src/components/SignUpModal/SignInUpModal.jsx:3]: done

File: src/components/SignUpModal/SignInUpModal.jsx (hunk 2 above)
Comment (StephanieKirshner) [reply src/components/SignUpModal/SignInUpModal.jsx]: done

File: src/App.test.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (jdickerson-banyan) [reply src/App.test.js:5]: This page is still showing pages when it shouldn't

File: src/components/SignUpModal/SignInUpModal.jsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+                    <br />
+                    <input id="submit" type="submit" value="Sign In" />
+                </div>}
+            </Content>
+        </ModalContainer>
+
+    )
+}
+
+export default SignUpModal;
```
Comment (jdickerson-banyan) [inline src/components/SignUpModal/SignInUpModal.jsx:40]: add an blank line at the end of this file

File: src/components/SignUpModal/style.js [hunk 10]
```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components';
```
Comment (jdickerson-banyan) [inline src/components/SignUpModal/style.js:1]: what is the { css } doing here?

File: src/components/SignUpModal/style.js [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+        margin-left: 8.5%;
+    }
+    #password {
+        margin-left: 10%;
+    }
+    #submit {
+        margin-left: 30%;
+        background-color: #9fb5d5;
+    }
+`;
```
Comment (jdickerson-banyan) [inline src/components/SignUpModal/style.js:33]: add a blank line to the end of this file

File: src/setupTests.js (hunk 3 above)
Comment (jdickerson-banyan) [inline src/setupTests.js:6]: this file should not be changed

**Review by jdickerson-banyan — CHANGES_REQUESTED** [review]

File: src/components/SignUpModal/SignInUpModal.jsx (hunk 9 above)
Comment (StephanieKirshner) [reply src/components/SignUpModal/SignInUpModal.jsx:40]: done

File: src/components/SignUpModal/style.js (hunk 11 above)
Comment (StephanieKirshner) [reply src/components/SignUpModal/style.js:33]: done

**Pull request closed without merging**